Database/Base/ShardBase.sql

```diff
@@ -313,251 +245,168 @@ CREATE TABLE `ace_position` (
+/*!50001 CREATE TABLE  `vw_teleport_location`(
+ `name` text ,
+ `landblock` int(10) unsigned ,
+ `posX` float ,
+ `posY` float ,
+ `posZ` float ,
+ `qW` float ,
+ `qX` float ,
+ `qY` float ,
+ `qZ` float 
+)*/;
+
+/*View structure for view vw_ace_character */
+
+/*!50001 DROP TABLE IF EXISTS `vw_ace_character` */;
+/*!50001 DROP VIEW IF EXISTS `vw_ace_character` */;
 
-DROP VIEW IF EXISTS vw_ace_character;
-CREATE VIEW vw_ace_character AS
-SELECT
-  ao.aceObjectId AS guid
-  , aopiidacc.propertyValue AS accountId
-  , aops.propertyValue AS NAME
-  , aopb.propertyValue AS deleted
-  , aopbi.propertyValue AS deleteTime
-  , ao.weenieClassId AS weenieClassId
-  , awc.weenieClassDescription AS weenieClassDescription
-  , ao.aceObjectDescriptionFlags AS aceObjectDescriptionFlags
-  , ao.physicsDescriptionFlag AS physicsDescriptionFlag
-  , ao.weenieHeaderFlags AS weenieHeaderFlags
-  , aopi.propertyValue AS itemType
-FROM
-  ace_object ao
-  JOIN ace_weenie_class awc
-    ON ao.weenieClassId = awc.weenieClassId
-  JOIN ace_object_properties_string aops
-    ON ao.aceObjectId = aops.aceObjectId
-    AND aops.strPropertyId = 1
-  JOIN ace_object_properties_bool aopb
-    ON ao.aceObjectId = aopb.aceObjectId
-    AND aopb.boolPropertyId = 9001
-  JOIN ace_object_properties_int aopi
-    ON ao.aceObjectId = aopi.aceObjectId
-    AND aopi.intPropertyId = 1
-  JOIN ace_object_properties_bigint aopbi
-    ON ao.aceObjectId = aopbi.aceObjectId
-    AND aopbi.bigIntPropertyId = 9001
-  JOIN ace_object_properties_iid aopiidacc
-    ON ao.aceObjectId = aopiidacc.aceObjectId
-    AND aopiidacc.iidPropertyId = 9001
+/*!50001 CREATE ALGORITHM=UNDEFINED SQL SECURITY DEFINER VIEW `vw_ace_character` AS select `ao`.`aceObjectId` AS `guid`,`aopiidacc`.`propertyValue` AS `accountId`,`aops`.`propertyValue` AS `NAME`,`aopb`.`propertyValue` AS `deleted`,`aopbi`.`propertyValue` AS `deleteTime`,`ao`.`weenieClassId` AS `weenieClassId`,`awc`.`weenieClassDescription` AS `weenieClassDescription`,`ao`.`aceObjectDescriptionFlags` AS `aceObjectDescriptionFlags`,`ao`.`physicsDescriptionFlag` AS `physicsDescriptionFlag`,`ao`.`weenieHeaderFlags` AS `weenieHeaderFlags`,`aopi`.`propertyValue` AS `itemType` from ((((((`ace_object` `ao` join `ace_weenie_class` `awc` on((`ao`.`weenieClassId` = `awc`.`weenieClassId`))) join `ace_object_properties_string` `aops` on(((`ao`.`aceObjectId` = `aops`.`aceObjectId`) and (`aops`.`strPropertyId` = 1)))) join `ace_object_properties_bool` `aopb` on(((`ao`.`aceObjectId` = `aopb`.`aceObjectId`) and (`aopb`.`boolPropertyId` = 9001)))) join `ace_object_properties_int` `aopi` on(((`ao`.`aceObjectId` = `aopi`.`aceObjectId`) and (`aopi`.`intPropertyId` = 1)))) join `ace_object_properties_bigint` `aopbi` on(((`ao`.`aceObjectId` = `aopbi`.`aceObjectId`) and (`aopbi`.`bigIntPropertyId` = 9001)))) join `ace_object_properties_iid` `aopiidacc` on(((`ao`.`aceObjectId` = `aopiidacc`.`aceObjectId`) and (`aopiidacc`.`iidPropertyId` = 9001)))) */;
```

Comment:
yea - I had hand edited it and it was blowing up everyone for some reason.   I can try again - let me know.   I hate the way this generates - but the sqlyog guys say that is part of mysql.   I think we are swimming upstream on this one....